axon-framework/axon-framework-commands/infrastructure.md

```diff
@@ -414,59 +376,242 @@ public DisruptorCommandBus commandBus(TransactionManager txManager, AxonConfigur
+The load factor defines the amount of load an Axon application would carry compared to other instances.
+For example, if you have a two machine set up, both with a load factor of 100, both will carry an equal amount of load.
 
-Two implementations of the `RoutingStrategy` are provided: the `MetaDataRoutingStrategy`, which uses a metadata property in the command message to find the routing key, and the `AnnotationRoutingStrategy`, which uses the `@TargetAggregateIdentifier` annotation on the Command Messages payload to extract the routing key. Obviously, you can also provide your own implementation.
+Increasing the load factor to 200 on both would still mean that both machines receive the same amount of load.
+This points out that the load factor will only change load amongst systems if the values are in equal.
+Doing so would make sense in a heterogeneous application landscape, where for the faster machines should deal with a bigger portion of command handling than the slower machines.
 
-By default, the `RoutingStrategy` implementations will throw an exception when no key can be resolved from a command message. This behavior can be altered by providing a `UnresolvedRoutingKeyPolicy` in the constructor of the `MetaDataRoutingStrategy` or `AnnotationRoutingStrategy`. There are three possible policies:
+The default load factor set for the distributed `CommandBus` implementations is 100.
+The configuration changes slightly per distributed implementation and as such will be covered in those sections.
 
-* `ERROR` - the default, and will cause an exception to be thrown when a Routing Key is not available
-* `RANDOM_KEY` - will return a random value when a \`routing key cannot be resolved from the command message.
+### Routing Strategy
 
-  This effectively means that those commands will be routed to a random segment of the command bus.
+Commands should be [routed consistently](../../architecture-overview/README.md#explicit-messaging) to the same application, especially those targeted towards a specific Aggregate.
+This ensures a single instance is in charge of the targeted aggregate, resolving the concurrent access issue and allowing for optimization like caching to work as designed.
+To component dealing with the consistent routing in an Axon application, is the `RoutingStrategy`.
 
-* `STATIC_KEY` - Will return a static key \(being "unresolved"\) for unresolved routing keys.
+The `RoutingStrategy` receives a `CommandMessage` and based on the message returns the routing key to use.
+Two commands with the same routing key will **always** be routed to the same segment, as long as there is no topology change in the distributed set-up.
 
-  This effectively means that all those commands will be routed to the same segment, as long as the configuration of segments does not change.
+At the moment, there are five implementations of the `RoutingStrategy`, where three of these are intended to be a fallback solution in case the routing key cannot be resolved:
```

Comment:
```suggestion
At the moment, there are five implementations of the `RoutingStrategy`. Three of these are intended to be a fallback solution, in case the routing key cannot be resolved:
```